src/Microsoft.DotNet.ImageBuilder/src/ViewModel/PlatformInfo.cs

```diff
@@ -138,23 +138,30 @@ public bool IsInternalFromImage(string fromImage)
 
         public string GetOSDisplayName()
         {
-            string displayName;
+            string displayName = null;
             string os = BaseOsVersion;
 
             if (Model.OS == OS.Windows)
             {
-                if (os.Contains("2016"))
+                string version = os.Split('-')[1];
+                if (os.StartsWith("nanoserver"))
                 {
-                    displayName = "Windows Server 2016";
+                    displayName = $"Windows Nano Server, version {version}";
```

Comment:
The brand name is just `Nano Server`.  
```suggestion
                    displayName = $"Nano Server, version {version}";
```

Reply:
Fixed